features/feed/src/main/java/com/android254/droidconKE2020/feed/ui/views/ShareFragment.kt

```diff
@@ -7,13 +7,10 @@ import android.view.ViewGroup
 import android.widget.Toast
```

Comment:
@michaelbukachi  Why the shift from View Binding to Kotlin Synthetics?

Reply:
I'll change it back. I'm trying to figure out what's wrong with UI tests. Doing process of elimination.

Reply:
Okay no worries then

Reply:
What's the progress on this, i want to make a PR on issue #54 and i want to pull the resources clean up changes

Reply:
Let me merge into develop. I'll still work on this branch though to work on the Ci bit